File: src/automatica.core/Automatica.WebNew/src/app/base/model/user/user2group.ts
import { BaseModel, Model, JsonFieldInfo, JsonProperty } from "../base-model";
import { UserGroup } from "./user-group";

@Model()
export class User2Group extends BaseModel {

    @JsonProperty()
    This2UserGroup: string;

    @JsonProperty()
    This2User: string;

    // @JsonProperty()
    // This2UserGroupNavigation: UserGroup;

    public typeInfo(): string {
        return "User2Group";
    }

    protected getJsonProperty(): Map<string, JsonFieldInfo> {
        return new Map<string, JsonFieldInfo>();
    }
}

File: src/automatica.core/Automatica.Core.Model/Models/User/User2Group.cs
using System;
namespace Automatica.Core.Model.Models.User
{
    public class User2Group : TypedObject
    {
        public Guid This2User { get; set; }
        public Guid This2UserGroup { get; set; }
        public User This2UserNavigation { get; set; }
        public UserGroup This2UserGroupNavigation { get; set; }
    }
}
